src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super Machine> machines, Machine value);
```

Comment:
The `fill` method should use the generic type `T` instead of `Machine` for the `value` parameter to ensure it works with any subclass of `Machine`. Change the parameter type to `T`.